mappings/net/minecraft/class_344.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_344 net/minecraft/class_344 ACC:PUBLIC
```

Comment:
What is this?

Reply:
thats where enigma crashed. must of not saved when i fixed it ill commit a change